frontend/src/errorHandling/ErrorHandler/ErrorHandler.tsx

```diff
@@ -0,0 +1,73 @@
+import { CloseOutlined } from '@ant-design/icons';
+import { Modal, Typography } from 'antd';
+import Button from 'antd-button-color';
+import { FC } from 'react';
+import { ErrorItem } from '.';
+import { CustomError, ErrorTypes } from '../utils';
+
+const { Text } = Typography;
+
+export interface IErrorHandlerProps {
+  errorsQueue: Array<CustomError>;
+  show: boolean;
+  dismiss: () => void;
+}
+
+const ErrorHandler: FC<IErrorHandlerProps> = ({ ...props }) => {
+  const { errorsQueue, dismiss, show } = props;
+
+  const mappedErrors = errorsQueue.map((e, i) => (
+    <ErrorItem key={i} item={e} />
+  ));
```

Comment:
Would put this directly inside the TSX

Reply:
i don't think it's a clean way, but if you prefer i remove the variable and i map everything inside the TSX